### ISSUE TITLE
TODO needs an update

### ISSUE THREAD

**Issue by rrrapha** [dyne/frei0r#76]
The [TODO](https://github.com/dyne/frei0r/blob/master/TODO) file contains outdated information and broken links. 
(Also, [NEWS](https://github.com/dyne/frei0r/blob/master/NEWS) is not very useful in its current form.)

**Comment by jaromil** [comment]
@ddennedy yes I saw it completely outdated so removed it from release, did not expect removing it would break autotools
